core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -847,10 +849,10 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
     withLibraryAttributeNamespaceCheck(operations.map(_.name)) {
       for {
         isCurator <- tryIsCurator(ctx.userInfo.userEmail)
-        workspace <- getWorkspaceContext(workspaceName) flatMap { workspace =>
+        workspace <- getV2WorkspaceContext(workspaceName) flatMap { workspace =>
```

Comment:
What the heck is a "library"? Is it dead code?

Reply:
I think this is a firecloud-ui feature, perhaps [this](https://portal.firecloud.org/#workspaces/anvil-datastorage/1000G-high-coverage-2019). I see no evidence of this in terra-ui.

Reply:
There's a bunch of [orch library endpoints](https://firecloud-orchestration.dsde-dev.broadinstitute.org/#/Library) as well. It looks like DUOS might use the feature, but I don't know much about what it does or how it works